src/main.rs

```diff
@@ -219,7 +219,7 @@ fn edit_workspace_dependency_table(
             }
         }
 
-        document.insert("workspace.dependencies", Item::Table(new_table));
+        document["workspace"]["dependencies"] = Item::Table(new_table);
```

Comment:
Ah good one. Yes was wondering why it did that. Tho does this panic if workspace table doesnt exist?

Reply:
It seems to create one, at least I haven't had a panic even when removing `["workspace"]` but it's a bit sus indeed.

Reply:
So this is something I will want to implement eventually but not yet had the time for. Detect if the table exist, if not create it, if it does, and has some entries already in it, we should add deps to the end of the table.